client/src/backend/components/subject/ListItem.js

```diff
@@ -21,7 +21,7 @@ export default class SubjectListItem extends PureComponent {
         <Link to={lh.link("backendSettingsSubject", subject.id)}>
           <header>
```

Comment:
Seems like this <header> tag is also semantically incorrect and should be removed.